tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -6,7 +6,9 @@
 import scipy.ndimage as ndi
 import torch
 import torchvision
+from matplotlib import path
 from skimage import exposure, filters
+from skimage.measure import regionprops
```

Comment:
Consider replacing with cv2 version for speed

Reply:
https://docs.opencv.org/3.4/de/d62/tutorial_bounding_rotated_ellipses.html

Reply:
https://www.pythonpool.com/cv2-boundingrect/

Reply:
Now using opencv.
https://github.com/TissueImageAnalytics/tiatoolbox/blob/3225a5835708a89cd5999385b406aa3d1a663fe8/tiatoolbox/tools/registration/wsi_registration.py#L617-L620